Src/VimCore/CommonOperations.fs

```diff
@@ -63,7 +63,7 @@ type internal CommonOperations
 
     do
         _textView.Caret.PositionChanged
-        |> Observable.subscribe (fun _ -> _maintainCaretColumn <- MaintainCaretColumn.None)
+        |> Observable.subscribe (fun _ -> this.MaintainCaretColumn <- MaintainCaretColumn.None)

```

Comment:
Stylistically, I think either way is fine because neither one is visible from outside of the type. But my motivation for changing it was so that I could set a breakpoint and find every caller who was modifying that mutable variable, versus setting half a dozen breakpoints, etc.